contact.html

```diff
@@ -1,6 +1,7 @@
 ---
 title: Contact
 layout: default
+meta-description: Feel free to talk with and participate in the IRCv3 Working Group on irc.freenode.net/#ircv3
```

Comment:
I think that maybe this should have the protocol and port, but as it's meta description, I am not so sure.